input/pagecontent/variance.md

```diff
@@ -0,0 +1,16 @@
+### Variance from AU Base
+This implementation guide has no variance (i.e. fully compliant) from AU Base FHIR Implementation Guide version 4.2.1-ci-build ([current](https://build.fhir.org/ig/hl7au/au-fhir-base/)).
+
+#### Additionally Profiled Resources
+This implementation guide profiles the following resources that are not profiled in AU Base: 
+
+#### Observation
```

Comment:
Can't leave this as a heading and have colon indicating a points list in the preceeding statement of "that are not profiled in AU Base:".

I'd prefer this to be a 1st level bullet point rather than heading but if you feel strongly it can be a heading just end the preceeding sentence in a bullet point.

Reply:
Removed heading & added first level bullet point.